src/wovnio/wovnphp/Headers.php

```diff
@@ -19,18 +19,22 @@ class Headers
     private $urlLang;
     private $query;
     private $browserLang;
+    private $cookies;
+    private $cookieLang;
 
     /**
      * Constructor of the Headers class
      *
      * @param array &$env Contains the _SERVER env variable
      * @param Store &$store The store containing user settings
+     * @param array $cookies &$cookies Contains the _COOKIE env variable
      * @return void
      */
-    public function __construct(&$env, &$store)
+    public function __construct(&$env, &$store, $cookies)
     {
         $this->env =& $env;
         $this->store =& $store;
+        $this->cookieLang = new CookieLang($cookies);
```

Comment:
`private $cookies` looks unused. You should also just replace the `$cookies` parameter with CookieLang.

Reply:
Private $cookies indeed is unused, apologies for all the unused stuff. My IDE's intelisense/code highlighting just refused to work properly somehow.

But I think it should be called $cookies, since it's the $_COOKIE super global we are effectively passing into the constructor there.

Reply:
I think that's fine. The CookieLang class is more of a wrapper anyway and could be called Cookies even 😉

Reply:
Yeah, after moving the bulk of logic into headers I agree.